recipe/meta.yaml

```
@@ -1,47 +1,129 @@
-{% set name = "mesaLib" %}
-{% set version = "17.2.0" %}
+{% set version = "25.1.0" %}
 
 package:
-  name: {{ name|lower }}
-  version: "{{ version }}"
+  name: mesalib
+  version: {{ version }}
 
 source:
-  fn: {{ name }}-{{ version }}.tar.gz
-  url: ftp://ftp.freedesktop.org/pub/mesa/mesa-{{ version }}.tar.gz
-  sha256: 9484ad96b4bb6cda5bbf1aef52dfa35183dc21aa6258a2991c245996c2fdaf85
+  url: https://archive.mesa3d.org/mesa-{{ version }}.tar.xz
+  sha256: b1c45888969ee5df997e2542654f735ab1b772924b442f3016d2293414c99c14
 
 build:
-  skip: True  # [win or py3k]
   number: 0
+  skip: true  # [win]
   track_features:
-      - mesalib
+    - mesalib
+  run_exports:
+    - {{ pin_subpackage('mesalib', max_pin='x.x') }}
 
 requirements:
   build:
-      - pthread-stubs
-      - pkgconfig
-      - gcc  # [linux]
-      - toolchain
-      - llvmdev
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - {{ compiler('rust') }}
+    - meson >=0.63.0
+    - ninja
+    - pyyaml
+    - mako
+    - packaging
+    - python 3.12
+    - flex
+    - bison
+    - pkg-config
+    - llvm
+    - llvmdev  # [build_platform != target_platform]
+    - glslang
+    - git  # [linux]
+  host:
+    - expat
+    - zlib
+    - zstd
+    - libdrm  # [linux]
+    - spirv-tools 2025.1
+    - llvm 17.0.6
+    - llvmdev 17.0.6
```

```suggestion
    - llvmdev 15
```
I am not sure of the state of llvmdev 17

---

Fixed